src/table/virtualized-table/TotalsCell.tsx

```diff
@@ -39,7 +40,7 @@ const TotalsCell = ({ index, style, data }: TotalsCellProps) => {
         borderTopWidth: totals.atBottom ? '1px' : '0px',
         borderBottomWidth: totals.atTop ? '1px' : '0px',
         padding: '4px 12px',
-        justifyContent: index === 0 ? 'left' : 'right',
+        justifyContent: index === 0 ? 'left' : align,
```

Comment:
Any reason why the first column doesn't use `align`. Looks like this in the old table
<img width="313" alt="Screenshot 2023-03-10 at 14 42 31" src="https://user-images.githubusercontent.com/9249820/224331326-6a0b3684-8f3d-4774-89dc-be3c76860eb3.png">

Reply:
ah, good question, I just assumed that I should leave it like it was, so no special reason from me

Reply:
ok then I think we should simply set it to `align`